source/settings.txt

```diff
@@ -12,8 +12,6 @@
    :depth: 1
    :class: singlecol
 
-.. include:: includes/fact-vsce-preview.rst
```

Comment:
Should we remove the https://github.com/mongodb/docs-mongodb-vscode/blob/master/source/includes/fact-vsce-preview.rst `fact-vsce-preview.rst` file now that it's unused as well?

Reply:
Done. Thanks @Anemy